### PR TITLE
Addition of docs exampling Attachments usage.

### PR DESCRIPTION
**Pull request by avelis**
Helps #4
Documentation will help developers understand how to utilize the
attachments argument and it’s expected data structure.

Finally, a link Slack’s API docs to guide the developer in learning
other configurations and custom usage.

### CODE REVIEW

File: django_slack/__init__.py [hunk 1]
```diff
@@ -69,6 +69,31 @@
 * **icon_url** -- URL to an image to use as the icon for this message. (eg. `http://lorempixel.com/48/48`)
 * **icon_emoji** -- Emoji to use as the icon for this message (eg. `:chart_with_upwards_trend:`). Overrides `icon_url`.
 
+Richly-formatted messages
+--------------------------
```
Comment (lamby) [inline django_slack/__init__.py]: missing newline afterwards

File: django_slack/__init__.py [hunk 2]
```diff
@@ -69,6 +69,31 @@
...
+You can send any number of richly-formatted messages as attachments with a given slack message.
```
Comment (lamby) [inline django_slack/__init__.py]: please word wrap line opening paragraphs in the file

Comment (lamby) [inline django_slack/__init__.py]: slack → Slack

File: django_slack/__init__.py [hunk 3]
```diff
@@ -69,6 +69,31 @@
...
+
+To send a message with an attachment.
+
+1. Assemble your attachments as follows::
```
Comment (lamby) [inline django_slack/__init__.py]: Should use the rst numbered list operator, not "1", etc.

File: django_slack/__init__.py [hunk 4]
```diff
@@ -69,6 +69,31 @@
...
+
+    attachments = [
+        {
+         'title': "Richly-formatted message title',
+         'text': 'Richly-formatted message body.'
```
Comment (lamby) [inline django_slack/__init__.py]: Trailing comma missing

File: django_slack/__init__.py [hunk 5]
```diff
@@ -69,6 +69,31 @@
...
+        }
```
Comment (lamby) [inline django_slack/__init__.py]: Trailing comma missing

File: django_slack/__init__.py [hunk 6]
```diff
@@ -69,6 +69,31 @@
...
+
+2. Pass in your attachments to `slack_message` as an optional argument::
+
+    from django_slack import slack_message
+    
+    slack_message('path/to/my_message.slack', {
+        'foo': Foo.objects.get(pk=17),
+    }, attachments)
+
+You can assemble and send any number of message objects within the `attachments` list.
```
Comment (lamby) [inline django_slack/__init__.py]: wrap here too please

**Comment by avelis** [comment]
@lamby Thanks for letting me know about documentation improvements. Commit: `8b317b0` should have you were looking for.

**Comment by lamby** [comment]
Thanks :)  I made two extra minor changes on master. Nice work.

**Comment by avelis** [comment]
@lamby You are most welcome. Thanks for merging the PR.

**Pull request merged**